src/App.tsx

```diff
@@ -1,9 +1,8 @@
 import * as React from 'react';
 import './styles/App.css';
-
 import { createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
-import userReducer from './reducers';
+import combineReducers from './reducers';
```

Comment:
Lets rename this to `rootReducer` since that is what it actually is. Using `combineReducers` could be confusing because it is a function in the `redux` library.

Reply:
index.ts
```
import { combineReducers } from 'redux';
import userReducer from './userReducer';
import projectReducer from './projectReducer';
import appReducer from './appReducer';

export default combineReducers({
  user: userReducer,
  projects: projectReducer,
  registerLoginWindow: appReducer
});
```

Reply:
App.tsx

```
import combineReducers from './reducers';

const store = createStore(
  combineReducers ,
  composeWithDevTools(applyMiddleware(thunk))
);
```

i thought we were incorporating combineReducers from index.ts , and putting it in store. So it is the function in the redux library? or am i thinking of this all wrong